okrs_2019.md

```diff
@@ -0,0 +1,229 @@
+# SourceCred 2019 OKRs
+
+This document contains SourceCred's objectives, initiatives, and metrics for
+2019, generated as part of Protocol Labs's annual planning process.
+For an overview of SourceCred itself, see [the overview](overview.md).
+
+The purpose of this document is to provide clarity around what the project's
+goals and priorities are, and accountability to see if we are making effective
+progress towards achieving those goals. However, this is not intended to lock
+us in to a particular goals or metrics—we know that over time, opportunities
+will present themselves, and some of the existing initiatives will prove
+ill-founded or infeasible. Also, our metrics have known deficiencies (e.g. we
+don't yet know how to measure the quality of a cred attribution).
+
+As such, this is intended to be a living document. We will update it both to
+record progress on our goals, and to modify and retarget our goals as we gain
+more information. Since this document is in Git, all revisions will be
+publically visible. If we remove or replace a goal, initiave, or metric, we'll
+clearly explain the rationale for doing so in the associated commit.
+
+
+# SourceCred Goals
+
+## Improve Attribution Quality
+
+SourceCred's core function is to produce cred attributions for open-source
+projects. Improving the quality of those attributions is the our highest
+priority. We want SourceCred to do a great job of assigning cred both to
+contributors, and also to specific contributions.
+
+SourceCred's current implementation is a promising start, but still leaves much
+to be desired. It does a decent job of assigning cred to contributors; however,
+it struggles with the greater challenge of valuing specific contributions.
```

Comment:
Is there a succinct way of showing off the data that enables you to make those twin claims (fair job of contributor cred, bad job of contribution cred)? Straw man is "Based on early product testing"

Reply:
It's based on early product testing :smile:.